### ISSUE TITLE
Mitigating Neural Network Overconfidence with Logit Normalization

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#262]
### 論文リンク
https://arxiv.org/abs/2205.09310
https://github.com/hongxin001/logitnorm_ood
https://www.youtube.com/watch?v=k63GLOljLAk

### 発表された年/専門誌・会議
ICML2022

## 概要
分布外入力を検出することは、実世界で機械学習モデルを安全に展開するために重要である。
しかし、ニューラルネットワークは、分布内入力と分布外入力の両方に対して異常に高い信頼度を生成する過信問題に苦しむことが知られている。

本研究では、ロジット正規化（LogitNorm）-クロスエントロピー損失に対する簡単な修正-により、この問題が緩和されることを示す。
この方法は、ロジットのノルムが学習中に増加し続け、過信的な出力につながるという分析から動機づけられている。
したがって、ロジットノルムの背後にある我々の重要なアイデアは、ネットワーク最適化の際に出力のノルムの影響を切り離すことである。

LogitNormで学習したニューラルネットワークは、分布内データと分布外データの信頼度スコアを非常に区別して出力することができる。
広範な実験により、LogitNormの優位性が実証され、一般的なベンチマークにおいて平均FPR95を最大42.30%削減することができた。

## 実験方法・結果
![Screenshot 2022-07-14 at 11 43 36 AM](https://user-images.githubusercontent.com/8721858/179022975-86e9815c-1ed0-4f01-abff-632a1c3ba1d2.png)
![Screenshot 2022-07-14 at 11 43 49 AM](https://user-images.githubusercontent.com/8721858/179022979-afdef701-c3b7-48d0-970d-8b76bb376f5e.png)
![Screenshot 2022-07-14 at 11 43 57 AM](https://user-images.githubusercontent.com/8721858/179022981-8875007a-e67b-410e-bce1-29391e5f9433.png)
